src/snowcli/cli/nativeapp/commands.py

```diff
@@ -40,20 +40,20 @@ def app_init(
     name: str = typer.Argument(
         ..., help="Name of the Native Apps project to be initiated."
     ),
-    git_url: str = typer.Option(
+    template_repo: str = typer.Option(
```

Comment:
One thing I discovered with cloning here:
If you provide an `https` url to a private repo instead of using the ssh url, the command line asks for your github username and password. If you enter your password, you get:
```
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
```
which says you need to enter your personal access token in the password prompt instead of your password. 
Should we mention this here in our CLI, or leave the logs as they are since they are descriptive enough to lead the user to the answer?

Reply:
I think it would be better to mention it in help